src/token_utilities.py

```diff
@@ -3,7 +3,7 @@
 from datetime import datetime, date
 from . import cache
 
-opaque_token_count = 0
+OPAQUE_TOKEN_COUNT_KEY = "opaque_token_count"
 
 def decode_access_token(token):
     _, payload, _ = token.split('.')
```

Comment:
You've already called the split method on token in the previous parent funtion. Why are you calling it again on the same data?